examples/cra-kitchen-sink/src/components/FlowTypeButton.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import React from 'react';
```

Comment:
Just out of curiosity, how do you get the linting error? I think following this setup should be enough btw:
https://github.com/gajus/eslint-plugin-flowtype#shareable-configurations